regulations/tasks.py

```diff
@@ -49,6 +49,10 @@ def submit_comment(self, comments, form_data, metadata_url):
             with html_to_pdf(html) as comment_pdf, \
                     build_attachments(files) as attachments:
                 pdf_url = cache_pdf(comment_pdf, metadata_url)
+
+                # Restore file position changed by cache_pdf
+                comment_pdf.seek(0)
+
```

Comment:
I thought of the very same option at first. On 2nd thoughts, it seemed strange to make it a contract of the _called_ function to restore the file position. It seemed more logical to have the caller manage it, as it is in charge of handing out the handles.